b2luigi/batch/processes/gbasf2.py

```diff
@@ -967,11 +968,12 @@ def setup_dirac_proxy():
     if not isinstance(lifetime, int) or lifetime <= 0:
         warnings.warn("Setting 'gbasf2_proxy_lifetime' should be a positive integer.", RuntimeWarning)
     hours = int(lifetime)
-    proxy_init_cmd = shlex.split(f"gb2_proxy_init -g belle -v {hours}:00")
+    proxy_init_cmd = shlex.split(f"gb2_proxy_init -g belle -v {hours}:00 --pwstdin")
 
     while True:
-        proc = run_with_gbasf2(proxy_init_cmd, ensure_proxy_initialized=False, capture_output=True, check=True)
-
+        pwd = getpass('Certificate password:')
```

Comment:
Very minor, but I would add a whitespace after the prompt, at least this seems to be the convention for other input prompts I see in other software. Though since we don't see the password anyway doesn't matter much.
```suggestion
        pwd = getpass('Certificate password: ')
```